assets/stylesheets/_base.scss

```diff
@@ -1,6 +1,6 @@
-$f1: 'bebas_neuebook', Georgia, serif;                     // title font
-$f2: 'callunaregular', Helvetica, sans-serif;       // text font
-$f3: "Courier New", Courier, monospace;  // code font
+$f1: 'bebas_neuebold', Georgia, serif;                     // title font
+$f2: 'abeezeeregular', Courier, monospace;      // text font
+$f3: 'callunaregular', Helvetica, sans-serif;  // code font
```

Comment:
For now this okay, but would prefer to see these mapped to vars that indicate the font-style: `mono`, `serif`, `sans-serif`. We should not have to dig to figure out what a variable means